samples/helloworld/static/greetings.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2015 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function GreetingsCtrl($scope, $http) {
+  $scope.greetings = [];
+  var button = document.getElementsByTagName('button')[0];
+  var api;
+ 
+  loadAPI(function() {
+    api = gapi.client.greetings;
+    $scope.refresh();
+ });
+
+  $scope.refresh = function() {
+    api.list({'limit':100}).execute(function(res) {
+      button.disabled = false;
```

Comment:
since you're using angular, you could just use `ng-disabled` property, no? so, expose something like `canAdd` boolean var on the `$scope` and then bind it to `ng-disabled` on the button.